catalog/ui/catalog-ui-search/src/main/webapp/component/property/property.js

```diff
@@ -195,6 +195,9 @@ define([
                 case 'AUTOCOMPLETE':
                     calculatedType = 'autocomplete';
                     break;
+                case 'ENUM':
```

Comment:
It should not be. Good catch. I'll give all the changes another once over for unneeded code